README.md

```diff
@@ -4,8 +4,29 @@
 ## Links
 [Figma](https://www.figma.com/file/vytKlKFg74ypSj6haevygK/Sunbird-RC-for-Hacktoberfest?node-id=0%3A1)
 
-[Issues](https://github.com/Sunbird-RC/vc-issuance/issues)
 
 [Swagger UI](https://sbrc-swagger.tiiny.site)
 
 [Discussions](https://github.com/Sunbird-RC/vc-issuance/discussions)
+
+## 🏍 To use , view or run the code :
+
+<img src="https://media.giphy.com/media/kdiLau77NE9Z8vxGSO/giphy.gif" width="40%"><br>
+
+* Clone / Download :
+
+       git clone https://github.com/your_username_/Project-Name.git
```

Comment:
you can add the actual url of the project

Reply:
Hey , if I put the actual url of the project , wouldn't it be wrong for the guy who wants to contribute

There are the scenarios :
1. If a guy needs to only see the project then  [ git clone actual url ] works
2. But if the guy needs to contribute , he first needs to fork and due to that the url of the project will change to 
<code>git clone https://github.com/your_username_/Project-Name.git</code>